utils/contributors.js

```diff
@@ -59,6 +59,32 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    {
+        handle_name: 'Garima-8604',
+        name: "Garima",
+        image: "C:\Users\HP\Downloads\Garima_idcard.jpeg",
```

Comment:
You have included a path from your local device for image so it won't work for other systems. 
Attach an image in `public` -> `images` folder and then use the concerned path.